data/registry_model/datatypes.py

```diff
@@ -353,7 +353,10 @@ def created_at(self):
         """
         When the manifest was created
         """
-        result = TagTable.get(TagTable.manifest_id == self.id).lifetime_start_ms
+        try:
+            result = TagTable.get(TagTable.manifest_id == self.id).lifetime_start_ms
+        except:
```

Comment:
You'll want to catch the specific exception `TagTable.DoesNotExist`. Any other exception is valid.